exercises/05-Challenge-GroceryList-part-4.jsx

```diff
@@ -124,4 +123,19 @@ class GroceryListItem extends React.Component {
   }
 }
 
+class GroceryListAdd extends React.Component {
+    constructor(props) {
+      super(props);
+    }
+
+    render() {
+      return (
+        <div>
+          <input className='new-item' type="text" onChange={this.props.inputChanged}/>
```

Comment:
The parent component (GroceryList) doesn't really need to be concerned with the text input changing or with storing the value of the text input when it changes.  That should be the responsibility of your new GroceryListAdd component.  The GroceryList component just needs to know when a new grocery list item needs to be added and the name of that grocery list item.